cdc/sink/codec/json.go

```diff
@@ -48,19 +49,54 @@ type column struct {
 func (c *column) FromSinkColumn(col *model.Column) {
 	c.Type = col.Type
 	c.Flag = col.Flag
-	c.Value = col.Value
 	if c.Flag.IsHandleKey() {
 		whereHandle := true
 		c.WhereHandle = &whereHandle
 	}
+	if col.Value == nil {
+		c.Value = nil
+		return
+	}
+	switch col.Type {
+	case mysql.TypeString, mysql.TypeVarString, mysql.TypeVarchar:
+		if s, ok := col.Value.(string); ok {
+			log.Info("show col "+s, zap.Any("c", c))
```

Comment:
Should it be removed?

Reply:
yes, but this pr is not ready yet...
I forget label a DNM